### PR TITLE
Fixes #266 add watermark size to chart settings

### LINKED ISSUE #266: Add watermark settings to chart settings

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
@Yuri05 This addresses the typo in previous PR plus also Watermark font size in settings

File: src/OSPSuite.UI/Extensions/ChartControlExtensions.cs [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
    {
-      public static void SetFontAndSizeSettings(this ChartControl chartControl, ChartFontAndSizeSettings fontAndSizeSettings,Size alternativeSize)
+      public static void SetFontAndSizeSettings(this ChartControl chartControl, ChartFontAndSizeSettings fontAndSizeSettings, Size alternativeSize)
       {
-         chartControl.Width = fontAndSizeSettings.ChartWidth.GetValueOrDefault() > 0 ? fontAndSizeSettings.ChartWidth.GetValueOrDefault() : alternativeSize.Width;
-         chartControl.Height = fontAndSizeSettings.ChartHeight.GetValueOrDefault() > 0 ? fontAndSizeSettings.ChartHeight.GetValueOrDefault() : alternativeSize.Height;
+         chartControl.Width = fontAndSizeSettings.ChartWidth.GetValueOrDefault(alternativeSize.Width);
+         chartControl.Height = fontAndSizeSettings.ChartHeight.GetValueOrDefault(alternativeSize.Height);
 
-         var fontTitle = new Font(fontAndSizeSettings.Fonts.FontFamilyName, fontAndSizeSettings.Fonts.TitleSize);
```
Comment (msevestre) [inline src/OSPSuite.UI/Extensions/ChartControlExtensions.cs]: I just don't like code repetition... ever :)

File: src/OSPSuite.UI/Extensions/ChartControlExtensions.cs [hunk 2]
```diff
@@ -30,5 +36,97 @@ public static void SetFontAndSizeSettings(this ChartControl chartControl, ChartF
             xyDiagram.GetAllAxesY().Each(axis => axis.Title.Font = fontAxis);
          }
       }
+
+      private const string WATERMARK_ANNOTATION = "WATERMARK_ANNOTATION";
```
Comment (msevestre) [inline src/OSPSuite.UI/Extensions/ChartControlExtensions.cs]: Moved those guys into the ChartControlExtension files. We had two files doing the same thing

File: src/OSPSuite.UI/Extensions/ChartControlExtensions.cs [hunk 3]
```diff
@@ -30,5 +36,97 @@ public static void SetFontAndSizeSettings(this ChartControl chartControl, ChartF
...
+
+      private static TextAnnotation createWatermarkAnnotation(ChartControl chartControl)
+      {
+         var annotation = chartControl.Annotations.AddTextAnnotation(WATERMARK_ANNOTATION);
+         annotation.ShapePosition = new FreePosition();
+         annotation.Angle = -45;
+         annotation.ConnectorStyle = AnnotationConnectorStyle.None;
+         annotation.BackColor = Color.Transparent;
+         annotation.Border.Visibility = DefaultBoolean.False;
+         annotation.TextColor = Constants.ChartFontOptions.DEFAULT_FONT_COLOR_WATERMARK;
```
Comment (msevestre) [inline src/OSPSuite.UI/Extensions/ChartControlExtensions.cs]: We do not have settigns for FONT COLOR Anywhere. I did not want to introduce somethign just foi a single font. In theory, we could also define FONT_COLOR in the export settings as part of the UI. Nice to have for another time

**Comment by msevestre** [comment]
@Yuri05 Please merge when you can. I'll update PK-Sim and MoBi after that.

File: src/OSPSuite.Core/Chart/ChartFonts.cs [hunk 4]
```diff
@@ -58,6 +61,12 @@ public string FontFamilyName
          set => SetProperty(ref _fontFamilyName, value);
       }
 
+      public virtual int WatermkarkSize
+      {
```
Comment (Yuri05) [inline src/OSPSuite.Core/Chart/ChartFonts.cs]: WatermkarkSize=>WatermarkSize

File: src/OSPSuite.Assets/UIConstants.cs [hunk 5]
```diff
@@ -182,7 +182,7 @@ public static class Captions
       public static string ShouldWatermarkBeUsedForChartExportToClipboard(string applicationName, string optionLocation)
       {
          var sb = new StringBuilder();
-         sb.AppendLine("The watermark feature was introduced to prevent accidental usage of plots in e.g. official reports or submissions.");
+         sb.AppendLine("The watermark feature was introduced to identify draft versions of plots.");
          sb.AppendLine();
```
Comment (Yuri05) [inline src/OSPSuite.Assets/UIConstants.cs]: I must say the previous message was pretty clear; the new one is not (for me)..

**Pull request merged**